### DISCUSSION TITLE
Elevation data in matrix API

### DISCUSSION THREAD

**Discussion by andersonluciano** [valhalla/valhalla#4250] [Q&A] [answered]
Hello everyone, we're implementing Valhalla here at the company, and a question that came up in one of the teams is whether elevation data is taken into account in the matrix API for the bicycle profile. Could you provide an answer to this question for us?

**Comment by nilsnolde** [comment, marked as answer]
Yes, unless specifically specified, all options are valid for most endpoints

**Comment by andersonluciano** [comment]
Tks @nilsnolde
